projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.virtualization.spec.ts

```diff
@@ -388,21 +388,21 @@ describe('IgxHierarchicalGrid Virtualization Custom Scenarios #hGrid', () => {
         const hierarchicalGrid = fixture.componentInstance.hgrid;
 
         const initialBodyWidth = hierarchicalGrid.tbody.nativeElement.offsetWidth;
-        expect(hierarchicalGrid.verticalScrollContainer.getScroll().parentElement.hidden).toBeTruthy();
+        expect(hierarchicalGrid.verticalScrollContainer.getScroll().parentElement.parentElement.hidden).toBeTruthy();
```

Comment:
You should probably shorthand this to something readable in the HierarchicalGridFunctions static methods